src/domain/ViewModel.ts

```diff
@@ -142,4 +144,8 @@ export class ViewModel<O extends Options = Options> extends EventEmitter<{change
     get navigation(): Navigation {
         return this._options.navigation;
     }
+
+    get history(): History {
+        return this._options.history;
+    }
```

Comment:
See https://github.com/vector-im/hydrogen-web/pull/749#discussion_r891899396 for why we're doing this now.